ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -125,6 +126,10 @@ message Commands {
   // Additional contracts used to resolve contract & contract key lookups.
   // Optional
   repeated DisclosedContract disclosed_contracts = 17;
+
+  // Must be a valid domain ID
+  // Optional: must be set for multi-domain API, must not be set otherwise. In the Ledger API 3.0 this will be mandatory .
```

Comment:
Same: do we want to say 3.0 instead of "multi-domain api" ?

Reply:
no, because 3.0 is an API version, and this will be introduced for 2.x, where the two APIs would live next to each other